connectors/src/api/webhooks/webhook_slack.ts

```diff
@@ -371,7 +377,86 @@ const _webhookSlackAPIHandler = async (
         }
         break;
       }
+      case "channel_created": {
+        const teamId = req.body.team_id;
+        const slackChannelId = req.body.event?.channel;
+        const slackConfiguration = await SlackConfigurationModel.findOne({
+          where: {
+            slackTeamId: teamId,
+          },
+        });
+        if (!slackConfiguration || !slackChannelId) {
+          return apiError(req, res, {
+            api_error: {
+              type: "slack_configuration_not_found",
+              message: `Slack configuration not found for teamId ${teamId}`,
+            },
+            status_code: 404,
+          });
+        }
+        const connectorId = slackConfiguration.connectorId;
```

Comment:
`nit` You can destructure variables in TS ([doc](https://www.typescriptlang.org/docs/handbook/variable-declarations.html#destructuring))
```suggestion
        const {connectorId} = slackConfiguration;
```